src/sim_init.h

```diff
@@ -78,6 +78,8 @@ class SimInit {
   void LoadDecomSched();
   void LoadNextIds();
 
+  void* LoadPreconditioner(std::string name);
+  ExchangeSolver* LoadGreedySolver(bool exclusive, std::set<std::string> tables);
```

Comment:
sigh, another reason to dislike [gcsg](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Line_Length), it accepst <=80 while PEP8 is [<80](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)